web-app/src/components/Orders/NationSummary.tsx

```diff
@@ -116,17 +107,17 @@ const NationSummary = ({
       </div>
```

Comment:
Need to wrap `disbandLabel` and `supplyCentersLabel` in `t()` to get translations to work